src/Components/Components/test/Forms/EditContextDataAnnotationsExtensionsTest.cs

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Forms;
+using System;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components.Tests.Forms
+{
+    public class EditContextDataAnnotationsExtensionsTest
+    {
+        [Fact]
+        public void CannotUseNullEditContext()
+        {
+            var editContext = (EditContext)null;
+            var ex = Assert.Throws<ArgumentNullException>(() => editContext.AddDataAnnotationsValidation());
+            Assert.Equal("editContext", ex.ParamName);
+        }
```

Comment:
I'm fine with us not writing these kinds of tests unless we think it's going to be especially error prone. We're going to have nullability support soon